tests/test_app.py

```diff
@@ -0,0 +1,54 @@
+# tests/test_app.py
+
+from app import app
+import unittest
+import os
+os.environ['TESTING'] = "true"
+
+
+class AppTestCase(unittest.TestCase):
+    def setUp(self):
+        self.client = app.test_client()
+
+    def test_home(self):
+        response = self.client.get("/")
+        assert response.status_code == 200
+        html = response.get_data(as_text=True)
+        assert """<h1 class="text-uppercase font-weight-bold">Welcome!</h1>""" in html
+
+        # TODO Add more tests relating to the home page
+        assert """<h1 class="project-title text-uppercase font-weight-bold">Projects</h1>""" in html
+
+
+    def test_timeline(self):
+        response = self.client.get("/api/timeline_post")
+        assert response.status_code == 200
+        assert response.is_json
+        json = response.get_json()
+        assert "timeline_posts" in json
+        assert len(json["timeline_posts"]) == len(json["timeline_posts"])
+        # TODO Add more tests relating to the /api/timeline_post GET and POST apis
+        # TODO Add more tests relating to the timeline page
```

Comment:
@MMAT30 you need to add TODO tasks:
Add more tests relating to the /api/timeline_post GET and POST apis
Add more tests relating to the timeline page